### DISCUSSION TITLE
Error updating? v1.2

### DISCUSSION THREAD

**Discussion by razidan** [alanbjohnston/CubeSatSim#349] [General]
Hi all,
I'm assembling the V1.2 (another one) and when I tried to run cubesatsim/update, I got this error
fatal: unable to access 'http://github.com/alanbjohnston/CubeSatSim.git/': Could not resolve host: github.com

Is this a problem?

Thanks,
Raz

**Comment by alanbjohnston** [comment]
Hi Raz,

The link is good - try pasting it into your browser. 

Most likely your Pi Zero isn’t connected to the internet. Try this command:

timeout 10 ping amazon.com

If you see messages such as "64 bytes received" then you are connected.  

If you aren’t, try the steps listed here for connecting your Pi Zero to the internet:

https://github.com/alanbjohnston/CubeSatSim/wiki/2.-Software-Install#logging-into-your-raspberry-pi

Alan

**Reply by razidan** [reply]
Yup! That was it!

Thank you!
